changelog/unreleased/enhancement-relative-date-tooltips

```diff
@@ -0,0 +1,6 @@
+Enhancement: Add tooltips to relative dates
+
+Relative dates like "1 day ago" now have a tooltip that shows the absolute date. This tooltip is a11y-compliant and can be reached via tab.
```

Comment:
Tooltips that get created with the directive `v-oc-tooltip` are aria-hidden by default, see https://github.com/owncloud/owncloud-design-system/blob/469a1c710b46ee8af6d46586ab5b3a41252894e8/src/directives/OcTooltip.js#L74

Afaik we made sure that anything that uses a `v-oc-tooltip` had proper accessible content on its own, via aria-label or aria-describedby. This needs a general decision:
a) make tooltips accessible (enforce tabindex, ideally via v-oc-tooltip directive, get rid of aria-hidden, think about other aria-props involved) or
b) keep tooltips aria-hidden, don't make them reachable via keyboard, make sure that accessible content exists which complements the tooltip content.

Reply:
Personally I'm fine with both. I mean technically, the tooltips have proper content already. Therefore option b) would be sufficient and easier to achieve for now.